app/helpers/application_helper.rb

```diff
@@ -18,7 +18,11 @@ def to_enhanced_options(data, &to_data_attrs)
 
   def register_form_with(*args, &block)
     options = args.extract_options!
-    defaults = { html: { novalidate: true, autocomplete: :off, spellcheck: false } }
+    defaults = { html: { novalidate: true,
+                         autocomplete: :off,
+                         "data-js-disable-browser-autofill": :on,
+                         spellcheck: false,
+                       } }
```

Comment:
It'll probably be happy with something like this:

```suggestion
    defaults = {
      html: {
        novalidate: true,
        autocomplete: :off,
        "data-js-disable-browser-autofill": :on,
        spellcheck: false,
      },
    }
```